src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -335,6 +335,31 @@ class WorkspaceApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  it should "delete all entities when deleting a workspace" in withTestDataApiServices { services =>
```

Comment:
Since deleting Method Configs and Submissions changed too, they should also have a test.